<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCheckParser</class>
 <widget class="QMainWindow" name="frmCheckParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проверка работы парсера</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gbSource">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Исходная запись</string>
    </property>
    <widget class="QPushButton" name="pbBegin">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>450</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Начало</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbLeft">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>450</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;==</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRight">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>450</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>==&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbEnd">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>450</y>
       <width>100</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Конец</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lwSource">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbParsed">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>511</width>
      <height>481</height>
     </rect>
    </property>
    <property name="title">
     <string>Новая запись</string>
    </property>
    <widget class="QListWidget" name="lwParsed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>491</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pbGetParsed">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>61</width>
      <height>441</height>
     </rect>
    </property>
    <property name="text">
     <string>==&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRecordNumber">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>510</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="pbGo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>500</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
